WordPress/WordPressTest/PeopleServiceTests.swift

```diff
@@ -0,0 +1,224 @@
+import XCTest
+import Nimble
+import OHHTTPStubs
+
+@testable import WordPress
+
+class PeopleServiceTests: CoreDataTestCase {
+
+    let siteID = 123
+    var service: PeopleService!
+
+    override func setUpWithError() throws {
+        super.setUp()
+
+        stub(condition: isHost("public-api.wordpress.com")) { request in
+            NSLog("Received an unexpected request: \(request)")
```

Comment:
`NSLog`? Any reason for not using `print`/`debugPrint`?

Is this log valuable outside the scope of this PR's development?

Reply:
Muscle memory, I think, is the reason I went with `NSLog` 😅 

I've changed this log to a test failure in https://github.com/wordpress-mobile/WordPress-iOS/pull/19947/commits/a6130e79886c9ca059f3fb8bb86b33303b97dc19